x-pack/plugins/enterprise_search/public/applications/analytics/components/analytics_collection_view/analytics_collection_view.tsx

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useEffect } from 'react';
+import { useParams } from 'react-router-dom';
+
+import { useActions, useValues } from 'kea';
+
+import {
+  EuiEmptyPrompt,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiSpacer,
+  EuiTitle,
+  EuiIcon,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import { generateEncodedPath } from '../../../shared/encode_path_params';
+import { KibanaLogic } from '../../../shared/kibana';
+import { EuiButtonTo } from '../../../shared/react_router_helpers';
+import { COLLECTION_CREATION_PATH, COLLECTION_VIEW_PATH } from '../../routes';
+
+import { EnterpriseSearchAnalyticsPageTemplate } from '../layout/page_template';
+
+import { AnalyticsCollectionIntegrate } from './analytics_collection_integrate';
+import { AnalyticsCollectionSettings } from './analytics_collection_settings';
+
+import { FetchAnalyticsCollectionLogic } from './fetch_analytics_collection_logic';
+
+export const collectionViewBreadcrumbs = [
+  i18n.translate('xpack.enterpriseSearch.analytics.collectionsView.breadcrumb', {
+    defaultMessage: 'View collection',
+  }),
+];
+
+export const AnalyticsCollectionView: React.FC = () => {
+  const { fetchAnalyticsCollection } = useActions(FetchAnalyticsCollectionLogic);
+  const { analyticsCollection, isLoading, hasNoAnalyticsCollection } = useValues(
+    FetchAnalyticsCollectionLogic
+  );
+  const { name, section } = useParams() as { name: string; section: string };
```

Comment:
suggestion: Avoid typecasting as much as possible. Instead, you can either type the const itself, or use a generic argument for useParams:
```
const { name, section } = useParams<{ name: string; section: string;}>;
```